circle.yml

```diff
@@ -0,0 +1,7 @@
+machine:
+  node:
+    version: 6.5.0
+test:
+  pre:
+    - npm install mocha --save-dev
+    - npm install expect.js --save-dev
```

Comment:
You don't need expect here. Simply install in your machine or add in your package.json.
Circle CI runs `npm install` for you. 

Use `npm install mocha -g`
You're not saving anything. The Circle runs on a container